docs/docsource/operation/Upgrading_XDM.rst

```diff
@@ -0,0 +1,28 @@
+.. _Upgrading XDM:
+
+Upgrading XDM
+=============
+
+#. Install the new ISO with:
+   
+   .. code::
+      
+      curl -u '$CREDS' https://eve.devsca.com/github/scality/zenko/artifacts/builds/github:scality:zenko:staging-2.0.0.r210210230152.7c4bbdc.pre-merge.00015938/zenko-2.0.0-alpha.3.iso -o zenko-2.0.0-alpha.3.3.iso
```

Comment:
This looks like a bunch of alpha and pre-release referents. When this is 2.0.0 GA, what will the targets be named?

Reply:
2.0.0-beta.x

Reply:
Use the |version| variable to replace the entire version number. Here, this should be: 
`zenko-|version|.iso `. Search the whole document and wherever you see 2.0.0-alpha.x or beta.y, replace it with |version|.